JS.md

```diff
@@ -205,6 +205,25 @@ to the Object `{ module: String(i.module_name), name: String(i.item_name), kind:
 
 Note: other fields like `signature` may be added in the future.
 
+### `WebAssembly.Module.userSections`
```

Comment:
Ugh, I hadn't noticed; it seems we call them *both* user-defined and unknown sections.  Seems like we should just go with one.  I actually like "unknown" better than "user defined"; it contrasts better with "known".  @rossberg-chromium ?